docs/src/architecture/index.md

```
@@ -0,0 +1 @@
+# Architecture
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)